aws/aws.go

```diff
@@ -0,0 +1,21 @@
+package aws
+
+// NS -
+func NS() *awsFuncs {
+	if af == nil {
```

Comment:
thanks, this seemed inelegant, but I didn't want to use `init()` - I'll replace with `sync.Once` 🙂